app/views/layouts/application.html.erb

```diff
@@ -30,6 +30,9 @@
           <li class="nav-item">
             <a class="nav-link" href="/dashboard">Profile</a>
           </li>
+          <li class="nav-item">
```

Comment:
We need to change these paths to look like the resource routes, ie `dashboard_path`.